package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
- With bugs, I went with the string to clean it up and make it consistent with the rest of the attributes, as per [npm's documentation](https://docs.npmjs.com/files/package.json#bugs), since we're not adding a dedicated email for bugs, but it may be a good idea to consider it.

- Directories, repository url and dependencies were changed automatically, probably in one of the many 'npm init' tests I tried, and I didn't notice until now. I'll revert them as they were and make sure that's what's producing those changes just in case it's 'npm version', which also commits to git.

- I added the keywords manually since [they serve as search terms](https://docs.npmjs.com/files/package.json#keywords) to look for the package in npm's website. Suggestions on more of them are very welcome, of course.